packages/fast-component-explorer/app/explorer.style.ts

```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
     },
     explorer: {},
     explorer_colorPicker: {
-        background: neutralFillRest(() => "#3B3B3B"),
+        background: neutralFillRest(neutralLayerL2),
         border: "none",
         width: "23px",
+        height: "22px",
         ...applyCornerRadius(),
         "&:hover": {
-            background: neutralFillHover(() => "#3B3B3B"),
+            background: neutralFillHover(neutralLayerL2),
         },
         "&:active": {
-            background: neutralFillActive(() => "#3B3B3B"),
+            background: neutralFillActive(neutralLayerL2),
         },
```

Comment:
nit / fyi, we can likely chain these selectors:

```
“&:hover, &:active”: {}
```

Reply:
the hover and active colors are different

Reply:
Ah - doh! I see the recipe is different. My mind went blank at neutralFill